test/useFormo.test-d.ts

```diff
@@ -4,34 +4,69 @@ import { either, taskEither } from "fp-ts";
 import { NonEmptyArray } from "fp-ts/NonEmptyArray";
 import { Option } from "fp-ts/Option";
 
-const { fieldProps, fieldErrors } = useFormo(
-  {
-    initialValues: {
-      name: "",
-      age: 23,
+{
+  const { fieldProps, fieldErrors } = useFormo(
+    {
+      initialValues: {
+        name: "",
+        age: 23,
+      },
+      fieldValidators: () => ({
+        name: validators.maxLength(10, false),
+        age: validators.validator((i: number) =>
+          i >= 18 ? either.right(i.toString()) : either.left(false)
+        ),
+      }),
     },
-    fieldValidators: () => ({
-      name: validators.maxLength(10, false),
-      age: validators.validator((i: number) =>
-        i >= 18 ? either.right(i.toString()) : either.left(false)
-      ),
-    }),
-  },
-  {
-    onSubmit: (values) => {
-      expectType<string>(values.name);
-      expectType<string>(values.age);
-      return taskEither.left(values);
+    {
+      onSubmit: (values) => {
+        expectType<string>(values.name);
+        expectType<string>(values.age);
+        return taskEither.left(values);
+      },
+    }
+  );
+
+  expectType<string>(fieldProps("name").value);
+  expectType<(v: string) => unknown>(fieldProps("name").onChange);
+  expectType<Option<NonEmptyArray<boolean>>>(fieldProps("name").issues);
+
+  expectType<number>(fieldProps("age").value);
+  expectType<(v: number) => unknown>(fieldProps("age").onChange);
+  expectType<Option<NonEmptyArray<boolean>>>(fieldProps("age").issues);
+
+  expectType<Record<"name" | "age", Option<NonEmptyArray<boolean>>>>(
+    fieldErrors
+  );
+}
+
+{
+  const { fieldProps, fieldErrors } = useFormo(
+    {
+      initialValues: {
+        name: "",
+        age: 23,
+      },
+      fieldValidators: () => ({}),
     },
-  }
-);
+    {
+      onSubmit: (values) => {
+        expectType<string>(values.name);
+        expectType<number>(values.age);
+        return taskEither.left(values);
+      },
+    }
+  );
 
-expectType<string>(fieldProps("name").value);
-expectType<(v: string) => unknown>(fieldProps("name").onChange);
-expectType<Option<NonEmptyArray<boolean>>>(fieldProps("name").issues);
+  expectType<string>(fieldProps("name").value);
+  expectType<(v: string) => unknown>(fieldProps("name").onChange);
+  expectType<Option<NonEmptyArray<boolean>>>(fieldProps("name").issues);
 
-expectType<number>(fieldProps("age").value);
-expectType<(v: number) => unknown>(fieldProps("age").onChange);
-expectType<Option<NonEmptyArray<boolean>>>(fieldProps("age").issues);
+  expectType<number>(fieldProps("age").value);
+  expectType<(v: number) => unknown>(fieldProps("age").onChange);
+  expectType<Option<NonEmptyArray<boolean>>>(fieldProps("age").issues);
 
-expectType<Record<"name" | "age", Option<NonEmptyArray<boolean>>>>(fieldErrors);
+  expectType<Record<"name" | "age", Option<NonEmptyArray<boolean>>>>(
+    fieldErrors
+  );
+}
```

Comment:
shouldn't we verify the intended type (`never`) is inferred for errors here?

Reply:
I left the check with `Option<NonEmptyArray<boolean>>` to understand if the type checker complains about it, but maybe we should directly check for type `never`.

Reply:
I see, maybe then we could keep both